packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/angular/load-app.md

```diff
@@ -0,0 +1,80 @@
+The first step for any application is to install or embed the Widget. You can link out to the Okta CDN or install locally through a package manager (for example, `npm`).
+
+### To use the CDN, include the following tags in your HTML:
+
+```html
+<!-- Latest CDN production JavaScript and CSS -->
+<script src="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/js/okta-sign-in.min.js" type="text/javascript"></script>
+<link href="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+```
+
+More information, including the latest published version, is available in the [Okta Sign-In Widget Documentation](https://github.com/okta/okta-signin-widget#using-the-okta-cdn).
+
+### npm
+
+```bash
+# Run this command in your project root folder.
+npm install @okta/okta-signin-widget@-=OKTA_REPLACE_WITH_WIDGET_VERSION=-
+```
+
+More information, including the latest published version, is available in the [Okta Sign-In Widget Documentation](https://github.com/okta/okta-signin-widget#using-the-npm-module).
+
+The sample application embeds the Sign-In Widget in the `login.components.ts` file:
+
+```javascript
+import { Component, OnInit } from '@angular/core';
+import { OktaAuth, Tokens } from '@okta/okta-auth-js';
+
+// @ts-ignore
+import * as OktaSignIn from '@okta/okta-signin-widget';
+import sampleConfig from '../app.config';
+
+const DEFAULT_ORIGINAL_URI = window.location.origin;
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+  signIn: any;
+  constructor(public oktaAuth: OktaAuth) {
+    this.signIn = new OktaSignIn({
+      /**
+       * Note: when using the Sign-In Widget for an OIDC flow, it still
+       * needs to be configured with the base URL for your Okta Org. Here
+       * we derive it from the given issuer for convenience.
+       */
+      baseUrl: sampleConfig.oidc.issuer.split('/oauth2')[0],
+      clientId: sampleConfig.oidc.clientId,
+      redirectUri: sampleConfig.oidc.redirectUri,
+      logo: 'assets/angular.svg',
+      i18n: {
+        en: {
+          'primaryauth.title': 'Sign in to Angular & Company',
+        },
+      },
+      authClient: oktaAuth,
+      useInteractionCodeFlow: sampleConfig.widget.useInteractionCodeFlow === 'true',
+    });
+  }
+```
+
+And references the Okta app configuration information in `app.config.js` or `env.js` and the `testenv` file:
```

Comment:
```suggestion
And references the Okta app configuration information in `app.config.js` or `env.js` file and the `testenv` file:
```

Reply:
They're all file here, so, it's a bit clunky if I mention that. How about: 

_And references the Okta app configuration information in `app.config.js` or in the 'env.js` and `testenv` files._